metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -403,6 +404,8 @@ def ensure_has_system_metadata(
     if event.systemMetadata is None:
         event.systemMetadata = SystemMetadataClass()
     metadata = event.systemMetadata
+    if metadata.lastObserved == 0:
+        metadata.lastObserved = int(time.time() * 1000)
```

Comment:
@hsheth2 suggest using `int(datetime.now(timezone.utc).timestamp() * 1000)` to ensure timestamp is in UTC

Reply:
`time.time()` already does epoch seconds